doc/changes.rst

```diff
@@ -1,6 +1,81 @@
 Changelog
 =========
 
+Release v0.2.1 (2020/XX/XX)
+---------------------------
+
+.. image:: https://zenodo.org/badge/DOI/10.5281/zenodo.4253459.svg
+   :alt: Digital Object Identifier for the Zenodo archive
+   :target: https://doi.org/10.5281/zenodo.4253459
+
+Highlights
+
+* 🎉 **Patch release with more tutorials and gallery examples!** 🎉
+* Support Python 3.9 (`#689 <https://github.com/GenericMappingTools/pygmt/pull/689>`__)
+* 📹 Add `Liam <https://github.com/liamtoney>`__'s `ROSES 2020 PyGMT talk <https://www.youtube.com/watch?v=SSIGJEe0BIk>`__ (`#643 <https://github.com/GenericMappingTools/pygmt/pull/643>`__)
+
+New Features
+
+* Wrap plot3d (`#471 <https://github.com/GenericMappingTools/pygmt/pull/471>`__)
+* Wrap grdfilter (`#616 <https://github.com/GenericMappingTools/pygmt/pull/616>`__)
+
+Enhancements
+
+* Allow np.object dtypes into virtualfile_from_vectors (`#684 <https://github.com/GenericMappingTools/pygmt/pull/684>`__)
+* Let plot() accept record-by-record transparency (`#626 <https://github.com/GenericMappingTools/pygmt/pull/626>`__)
+* Refactor info to allow datetime inputs from xarray.Dataset and pandas.DataFrame tables (`#619 <https://github.com/GenericMappingTools/pygmt/pull/619>`__)
+
+Tutorials & Gallery
+
+* Add tutorial for pygmt.config (`#482 <https://github.com/GenericMappingTools/pygmt/pull/482>`__)
+* Add an example for different line styles (#604, #664)
```

Comment:
```suggestion
* Add an example for different line styles (`#604 <https://github.com/GenericMappingTools/pygmt/pull/604>`__, `#664 <https://github.com/GenericMappingTools/pygmt/pull/664>`__)
```

Reply:
Nice catch. I believe it also means we should update our `sed` command.